### PR TITLE
Hare automation assertions

### PR DESCRIPTION
**Pull request by gavraz**
Added assertions to hare automation

### CODE REVIEW

File: tests/hare/assert_hare.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+def assert_all(curr_idx, ns):
+    total = testconfig['bootstrap']['replicas'] + testconfig['client']['replicas']
+
+    # assert_result
+    lst = query_hare_output_set(curr_idx, ns)
+    assert total == len(lst)
+    assert validate(lst)
+
+    # assert round 1
```
Comment (antonlerner) [inline tests/hare/assert_hare.py]: I'd revise comment to - assert that round 1 had enough whatever messages

Comment (gavraz) [reply tests/hare/assert_hare.py]: Fixed

File: tests/queries.py [hunk 2]
```diff
@@ -280,3 +282,30 @@ def find_missing(indx, namespace, client_po_name, fields, min=1):
     print("Total hits: {0}".format(len(hits)))
     print("Missing count {0}".format(len(miss)))
     print(miss)
+
+
+def query_hare_output_set(indx, ns):
+    hits = query_message(indx, ns, ns, {'M': 'Consensus process terminated'}, True)
+    lst = []
```
Comment (antonlerner) [inline tests/queries.py]: real men use one liners XD

Comment (antonlerner) [inline tests/queries.py]: real men use one liners XD

**Review by antonlerner — APPROVED** [review]

**Review by antonlerner — APPROVED** [review]

**Pull request merged**